lua/neogit/status.lua

```diff
@@ -1043,55 +1078,82 @@ local cmd_func_map = function()
       a.util.scheduler()
       local section, item = get_current_section_item()
 
-      if item and section then
-        if section.name == "unstaged" or section.name == "staged" or section.name == "untracked" then
-          local path = item.name
-          local hunk, hunk_lines = get_current_hunk_of_item(item)
-          local cursor_row, cursor_col
-          if hunk then
-            cursor_row, cursor_col = unpack(vim.api.nvim_win_get_cursor(0))
-          end
+      if section then
```

Comment:
Could this part be moved into a separate function now that it is even larger?

Reply:
👍